cmake/CMakeLists.txt

```diff
@@ -28,6 +33,6 @@ write_basic_package_version_file(
 install(FILES "${PROJECT_BINARY_DIR}/cmake/SerialboxConfigVersion.cmake" DESTINATION cmake)
 
 ## This is a hack (use some variables which are set in SerialboxConfig.cmake.in):
-include( ${PROJECT_BINARY_DIR}/cmake/SerialboxConfig.cmake )
-configure_file(${CMAKE_SOURCE_DIR}/cmake/SerialboxConfig.sh.in "${PROJECT_BINARY_DIR}/cmake/SerialboxConfig.sh" @ONLY)
-install(FILES "${PROJECT_BINARY_DIR}/cmake/SerialboxConfig.sh" DESTINATION cmake)
+#include( ${PROJECT_BINARY_DIR}/cmake/SerialboxConfig.cmake )

```

Comment:
This is a hack, I dont even know what it does.... But I think is not ok, SerialboxConfig.cmake is only for other project to load after installation